execution/executor/src/block_executor/mod.rs

```diff
@@ -259,12 +265,34 @@ where
                             "Injected error in vm_execute_block"
                         )))
                     });
-                    V::execute_transaction_block(
+
+                    self.global_cache_manager
+                        .mark_block_execution_start(&state_view, Some(parent_block_id))
```

Comment:
is the point that execute_transaction_block doesn't have parent_block_id? maybe we should pass it there?